modules/common/condition/conditions.go

```diff
@@ -55,6 +55,9 @@ const (
 
 	// CronJobReadyCondition Status=True condition when cron jobs created ok.
 	CronJobReadyCondition Type = "CronJobReady"
+
+	// AnsibleEECondition Status=True condition when the AnsibleEE run has been created ok
+	AnsibleEECondition Type = "AnsibleEEReady"
```

Comment:
Is this a condition that is set/checked across multiple operators?  Just asking in case it isn't, in which case it would make more sense to just have it live in the operator that uses it.

Reply:
For the moment it is used in just one operator (ceilometer-operator), but as ansibleee objects are going to be created by a number of different operators, it seems reasonable to expect that other operators also have this need.

Reply:
Ack